src/main/java/com/launchdarkly/client/RedisFeatureStore.java

```diff
@@ -87,6 +108,24 @@ public RedisFeatureStore(URI uri, String prefix, long cacheTimeSecs, JedisPoolCo
   }
 
   /**
+   * Creates a new store instance that connects to Redis based on the provided {@link RedisFeatureStoreBuilder}.
+   *
+   * See the {@link RedisFeatureStoreBuilder} for information on available configuration options and what they do.
+   *
+   * @param builder the store configuration
+   */
+  public RedisFeatureStore(RedisFeatureStoreBuilder builder) {
```

Comment:
Sure, went with `protected` in case there is a need to subclass this in the future. How does that sound?